app/src/main/res/layout/organiser_fragment_profile.xml

```diff
@@ -43,203 +44,339 @@
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintHorizontal_bias="0.255"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@+id/O_P_textEmail" />
+
+            <EditText
+                android:id="@+id/O_P_textPhone"
+                android:layout_width="wrap_content"
+                android:layout_height="48dp"
+                android:ems="13"
+                android:focusable="false"
+                android:focusableInTouchMode="false"
+                android:fontFamily="@font/roboto_mono_medium"
+                android:hint="123 456 7890"
+                android:inputType="phone"
+                android:textColor="@color/dark_purple"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintHorizontal_bias="0.503"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@+id/O_P_phone" />
+
+            <TextView
+                android:id="@+id/O_P_email"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="12dp"
+                android:fontFamily="@font/roboto_mono_medium"
+                android:text="Email:"
+                android:textColor="@color/dark_purple"
+                android:textSize="20sp"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintHorizontal_bias="0.255"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@+id/O_P_textPronouns" />
+
+            <EditText
+                android:id="@+id/O_P_textEmail"
+                android:layout_width="wrap_content"
+                android:layout_height="48dp"
+                android:ems="13"
+                android:focusable="false"
+                android:focusableInTouchMode="false"
+                android:fontFamily="@font/roboto_mono_medium"
+                android:hint="moodeng@ualberta.ca"
+                android:inputType="textEmailAddress"
+                android:textColor="@color/dark_purple"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintHorizontal_bias="0.503"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@+id/O_P_email" />
+
+            <TextView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="4dp"
+                android:layout_marginEnd="228dp"
+                android:text="⭐Organiser"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintTop_toBottomOf="@+id/O_profile_image" />
+
+            <TextView
+                android:id="@+id/O_profile_facility"
```

Comment:
It seems like the naming convention for ids changed here to have "profile" instead of using "p". For consistency it would make sense to change the earlier ones to match with using the full word.